paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -192,9 +189,6 @@ internal class PaymentSheetViewModel @Inject internal constructor(
         }
 
         eventReporter.onInit(config)
-        if (googlePayLauncherConfig == null) {
-            savedStateHandle[SAVE_GOOGLE_PAY_STATE] = GooglePayState.NotAvailable
-        }
```

Comment:
Not necessary, as the value will be set once `handlePaymentSheetStateLoaded()` is called.